jenkins/opensearch/integ-test.jenkinsfile

```diff
@@ -106,12 +111,21 @@ pipeline {
                                                 // Due to docker is mounting the agent directory so it can communicated with the agent
                                                 // This sometimes causes the workspace to retain last run test-results and ends with build failures
                                                 // https://github.com/opensearch-project/opensearch-build/blob/6ed1ce3c583233eae4fe1027969d778cfc7660f7/src/test_workflow/test_recorder/test_recorder.py#L99
-                                                sh("rm -rf test-results")
+                                                sh("rm -rf test-results ${WORKSPACE}/${distribution} && echo sleep ${wait_seconds} seconds && sleep ${wait_seconds}")
+                                                echo "Downloading from S3: ${artifactPath}"
+                                                downloadFromS3(
+                                                    destPath: "$WORKSPACE/artifacts",
+                                                    bucket: "${ARTIFACT_BUCKET_NAME}",
+                                                    path: "${artifactPath}/",
+                                                    force: true
+                                                )
+                                                sh("mv -v $WORKSPACE/artifacts/${artifactPath} $WORKSPACE")
```

Comment:
NIT: may be this has to be considered for future, since `mv` can add more time
@peterzhuamazon mentioned this is a limitation with with using [s3Download](https://github.com/opensearch-project/opensearch-build/blob/main/vars/downloadFromS3.groovy#L4), it will end up downloading the entire prefix.